docs/overriding_agg_fn.rst

```diff
@@ -37,6 +37,11 @@ Choose from the following predefined aggregation functions:
 - ``openfl.component.aggregation_functions.YogiAdaptiveAggregation``
 
 
+.. _adaptive_aggregation_functions:
+
+Adaptive Aggregation Functions
+^^^^^^^^^^^^^^^
```

Comment:
It will be warning "WARNING: Title underline too short." The length of this line should be equal to the previous one.